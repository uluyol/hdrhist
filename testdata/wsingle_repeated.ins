0
1
123
123
123
641
2
---
