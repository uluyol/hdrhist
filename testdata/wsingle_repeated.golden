0.000,0.001,0.001,HISTFAAAACR42pNpmSzMwMDAwQABzFCaEUoz2X+ASjC9Z2Tr5GACAFbdBBI=
